### PR TITLE
Docs for OpenTelemetry traces and metrics

### PR DESCRIPTION
**Pull request by mikeminutillo**

### CODE REVIEW

File: nservicebus/operations/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: OpenTelemetry
+summary: Observing NServiceBus endpoints via OpenTelemetry
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:3]:
```suggestion
summary: Observability of NServiceBus endpoints with OpenTelemetry
```

File: nservicebus/operations/opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+component: core
+reviewed: 2022-06-29
+---
+
+NServiceBus version 8 and above provides diagnostic trace and meter information via [OpenTelemetry](https://opentelemetry.io/docs/instrumentation/net/).
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:8]:
```suggestion
NServiceBus version 8 and above supports [OpenTelemetry](https://opentelemetry.io/docs/instrumentation/net/) through traces, metrics, and logging.
```

File: nservicebus/operations/opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+```csharp
+using var traceProvider = Sdk.CreateTracerProviderBuilder()
+    // Add all NServiceBus sources
+    .AddSource("NServiceBus.*")
+    // Add an exporter
+    .AddConsoleExporter()
+    .Build();
+```
+
+See samples for instructions on how to send trace information to different tools.
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:23]: Should this link to something?

File: nservicebus/operations/opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Meters
+
+NServiceBus endpoints can be configured to expose OpenTelemetry meters related to message processing. To capture meter information, add the `NServiceBus.Core` meter source to the OpenTelemetry configuration:
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:27]:
```suggestion
NServiceBus endpoints can be configured to expose metrics related to message processing. To capture meter information, add the `NServiceBus.Core` meter source to the OpenTelemetry configuration:
```

File: nservicebus/operations/opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+```csharp
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    // All all NServiceBus meters
+    .AddMeter("NServiceBus.*")
+    // Add an exporter
+    .AddConsoleExporter()
+    .Build();
+```
+
+Once the source has been added, enable OpenTelemetry meterics on the NServiceBus endpoint configuration:
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:38]:
```suggestion
Once the source has been added, enable OpenTelemetry metrics on the NServiceBus endpoint configuration:
```

File: nservicebus/operations/opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+```csharp
+endpointConfiguration.EnableOpenTelemetryMetrics();
+```
+
```
Comment (lailabougria) [inline nservicebus/operations/opentelemetry.md:43]:
```suggestion

## Logging

NServiceBus already supports logging out of the box. To collect OpenTelemetry-compatible logging in NServiceBus endpoints, it's possible to configure the endpoint to connect traces and logging when using Microsoft.Extensions.Logging.

TODO: Link to sample in the other PR.
```

File: nservicebus/operations/opentelemetry.md (hunk 3 above)
Comment (mikeminutillo) [reply nservicebus/operations/opentelemetry.md]: Yup. I fixed that.

File: samples/open-telemetry/logging/sample.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
+---
+title: Connecting OpenTelemetry traces and logs
+summary: How to connect OpenTelemetry traces to logs
+reviewed: 2022-06-27
+component: Extensions.Logging
```
Comment (timbussmann) [inline samples/open-telemetry/logging/sample.md:5]: should this be just connected to core as the component?

**Review by timbussmann — COMMENTED** [review]
updated to beta 5

**Pull request merged**